Kernel/Thread.h

```diff
@@ -283,6 +284,7 @@ class Thread {
     bool has_unmasked_pending_signals() const;
     void terminate_due_to_signal(u8 signal);
     bool should_ignore_signal(u8 signal) const;
+    bool has_signal_handler(u8 signal);
```

Comment:
This function should be const since it does not modify the thread state in any way.
```suggestion
    bool has_signal_handler(u8 signal) const;
```